firoplugin/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java-library'
+}
+version = "1.0"
+repositories {
+    // Use 'jcenter' for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
+    mavenCentral()
+    maven {
+        url = 'https://www.hec.usace.army.mil/nexus/repository/maven-public'
+    }
+}
+dependencies {
+    implementation project(":FIRO_TSEnsembles")
+    implementation 'mil.army.usace.hec:hec-core:6.0-20210210.230316-12'
+    implementation files("C:\\Temp\\hec-externalplugin\\build\\libs\\hec-externaldata-1.0.0.jar")
+    //implementation 'mil.army.usace.hec:hec-externaldata:1.0.1@jar'
```

Comment:
This dependency , hec-externaldata, isn't in he right format. -- building with Java 11, but we need Java 8.

Reply:
fixed